packages/store/src/Store.sol

```diff
@@ -2,21 +2,22 @@
 pragma solidity >=0.8.24;
 
 import { STORE_VERSION } from "./version.sol";
-import { IStoreData } from "./IStoreData.sol";
+import { StoreWrite } from "./StoreWrite.sol";
 import { StoreRead } from "./StoreRead.sol";
 import { StoreCore } from "./StoreCore.sol";
 import { IStoreEvents } from "./IStoreEvents.sol";
 
 /**
- * @title Store Data Contract
+ * @title Store Contract
  * @author MUD (https://mud.dev) by Lattice (https://lattice.xyz)
  * @notice This contract integrates the core storage functionalities and provides an interface for data storage.
  * @dev This abstract contract initializes `StoreCore`, implements `storeVersion`, and read methods,
  * but not write methods.
+ * Note that `Store` does not implement `IStore`, because we want `IWorldKernel` to inherit `IStore` but it does not implement all the Store methods.
```

Comment:
This comment is a bit confusing to me. Can you elaborate? Would the following be accurate:
```suggestion
 * Note that `Store` does not implement `IStore`, because the public write methods require access control that needs to be implemented by the consumer.
```